jekyll/_cci2/building-docker-images.md

```diff
@@ -118,7 +118,30 @@ If your job requires a specific docker version, you can set it as a `version` at
           version: 18.06.0-ce
 ```
 
-CircleCI supports multiple versions of Docker and defaults to using `docker-17.09.0-ce`. Consult the [Stable releases](https://download.docker.com/linux/static/stable/x86_64/) or [Edge releases](https://download.docker.com/linux/static/edge/x86_64/) for the full list of supported versions.
+CircleCI supports multiple versions of Docker and defaults to using `docker-17.09.0-ce`. Following are the supported stable and edge versions:
```

Comment:
We should drop the `docker-` prefix from all of these examples, that’s not needed when the user specifies the version in their config - i’m not sure why it was there before either.

I’ll double-check this tomorrow.

Reply:
thanks, I removed those (the example didn't include docker- prefix so I'm sure you are correct